### PR TITLE
Doc minor improvement

### PR DESCRIPTION
**Pull request by lpinsivy**
<h2> Description </h2>

Improve FR & EN documentation about:

- life cycle
- installation
- update
- migrate
- add a new server on platform

<h2> Type of change </h2>

Please delete options that are not relevant.

- [ ] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [x] Updating documentation (missing information, typo...)

<h2> Target serie </h2>

- [x] 18.10.x
- [x] 19.04.x (master)

<h2> How this pull request can be tested ? </h2>

Check spelling and content

<h2> Checklist </h2>

<h5> Community contributors & Centreon team </h5>

- [x] I followed the **coding style guidelines** provided by Centreon
- [x] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [x] I have commented my code, especially **hard-to-understand areas** of the PR.
- [x] I have made corresponding changes to the **documentation**.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

<h5> Centreon team only </h5>

- [x] I have made sure that the **unit tests** related to the story are successful.
- [x] I have made sure that **unit tests covers 80%** of the code written for the story.
- [x] I have made sure that **acceptance tests** related to the story are successful (**local and CI**)

### CODE REVIEW

File: doc/fr/installation/from_VM.rst [hunk 1]
```diff
@@ -61,6 +65,12 @@ l’utilisateur ‘centreon’ ::
     # su - centreon
     $ ssh-keygen -t rsa
 
+.. note::
+    Appuyez sur la touche *entrée* quand il vous sera demandé de saisir un
+    fichier pour enregistrer la clé. Laissez le mot de passe vide si vous le
```
Comment (sc979) [inline doc/fr/installation/from_VM.rst]: mot de passe ou phrase de récupération

File: doc/en/installation/from_VM.rst [hunk 2]
```diff
@@ -66,6 +70,12 @@ Copy this key to the new server: ::
 
 The password of the **centreon** user is *centreon*. It can be easily changed using the **passwd** command.
 
+.. note::
+    Hit enter when it prompts for a file to save the key to use the default,
```
Comment (ganoze) [inline doc/en/installation/from_VM.rst]: ...default location... would be more clear.

File: doc/fr/administration_guide/poller/install_poller_iso.rst [hunk 3]
```diff
@@ -12,8 +12,8 @@ Central réalisé à partir du fichier ISO de Centreon.
 .. note::
     Référez-vous à la documentation :ref:`installation<installisoel7>`.
 
-A la question **Which server type would you like to install?** il faut choisir
-l’option **Poller server**.
+A lieu de sélectionner l'option **Central with database**, il faut choisir
```
Comment (ganoze) [inline doc/fr/administration_guide/poller/install_poller_iso.rst]: Au lieu...

File: doc/en/administration_guide/poller/install_poller_iso.rst [hunk 4]
```diff
@@ -12,8 +12,9 @@ from a Centreon ISO file.
 .. note::
     Refer to the documentation: :ref:`installation<installisoel7>`
 
-In response to the question **Which server type would you like to install?** choose the
-option **Poller server**.
+Instead of choosing **Central with database**, you will select **Poller** and
+click **Done**. Please see the sizing guidelines for the poller server located
+`here <https://documentation.centreon.com/docs/centreon/en/latest/installation/prerequisites.html>`_.
```
[the remark this answers is not in the document]
Comment (lpinsivy) [reply doc/en/administration_guide/poller/install_poller_iso.rst]: no because here must be clickable

**Pull request merged**